### ISSUE TITLE
ExplainerDashboard not opening

### ISSUE THREAD

**Issue by raedbsili1991** [oegedijk/explainerdashboard#311]
Hello, 

I have executed this line of code that used to work in my previous virtual enrivorenement 

`ExplainerDashboard(ClassifierExplainer(RandomForestClassifier().fit(X_train, y_train), X_test, y_test)).run()`

I have got this error 

![Image](https://github.com/user-attachments/assets/a1bf991f-657e-4fb1-b764-1740a2f13c20)

Can someone please help me ? 

I tried all sort of re installing of explainerdashboard and as well plotly. 
I tried also all sort of upgrade/downgrade ...

**Comment by Metalkiler** [comment]
What version is your plotly, dash and dash-bootstrap-components? I had to downgrade those packages to the specfics versions:

pip install dash-bootstrap-components==1.7.1
pip install dash==2.18.2 
pip install plotly==5.5.0

**Comment by drewdstat** [comment]
It shouldn't be too difficult for the package maintainer to fix these issues. 

The first issue is line 360 in the explainer_plots.py script where the plotly_precision_plot function is being defined. This line calls to a deprecated `tickfont` argument (the list of current arguments can be found here: https://plotly.com/python-api-reference/generated/plotly.graph_objects.Layout.html). One could simply replace this line with `color="rgb(148, 103, 189)",` to fix that issue, though that changes the color for more than just the axis title. I'm not sure of the correct argument for just assigning the title font color in the newer versions of plotly.

The second issue is in dashboards.py in line 187 where the class ExplainerTabsLayout is having its layout defined and there is a deprecated call to an argument 'right' that no longer exists. I think replacing this line with `align_end=True,` would do the intended alignment and avoid an error.

That same script on line 1278 has an error where it calls `app.run_server`, which is deprecated and should instead be `app.run`. The same error is also on lines 1299, 1306, 2706, and 2710.

I was able to get the following toy example to run after making the aforementioned changes to the code, but I'm sure there's other errors that will pop up having to do with deprecated arguments intended for older versions of plotly or dash:

```
import shap
from sklearn.ensemble import RandomForestClassifier

from explainerdashboard import ClassifierExplainer, ExplainerDashboard
from explainerdashboard.datasets import titanic_survive

X_train, y_train, X_test, y_test = titanic_survive()
model = RandomForestClassifier(n_estimators=200, max_depth=5, random_state=42)
model.fit(X_train, y_train)
explainer = ClassifierExplainer(model, X_test, y_test)
dashboard = ExplainerDashboard(explainer, title="Titanic Survival Prediction")
dashboard.run()
```

This was on Python 3.10.12 with plotly version 6.1.2, dash version 3.0.4, and dash-bootstrap-components version 2.0.3.
